src/main/java/com/example/petbuddybackend/middleware/interceptor/ValidChatRoomAccessInterceptor.java

```diff
@@ -17,9 +17,12 @@
 @RequiredArgsConstructor
 public class ValidChatRoomAccessInterceptor implements ChannelInterceptor {
 
-    @Value("${url.chat.topic.base}")
+    @Value("${url.chat.topic.subscribe-prefix}")
     private String URL_CHAT_TOPIC_BASE;
 
+    @Value("${url.chat.topic.chat-id-pos}")
+    private int CHAT_ID_INDEX_IN_TOPIC_URL;
+
     @Value("${header-name.role}")
     private String ROLE_HEADER_NAME;
 
```

Comment:
a w handleMessageTopic napewno chodzi o taki warunek:
        if(!StompCommand.SUBSCRIBE.equals(command) && !StompCommand.SEND.equals(command)) {
            return;
        }
        ?
        
Bo wtedy jesli bedzie jedno z tych 2 to pojdzie dalej, a chyba nigdy command nie bedzie rowne 2 roznym wartością?

Reply:
> Bo wtedy jesli bedzie jedno z tych 2 to pojdzie dalej, a chyba nigdy command nie bedzie rowne 2 roznym wartością?

Jeżeli będzie `SUBSCRIBE` lub `SEND` to pójdzie dalej. Jeżeli będą oba (chociaż w tym kodzie się nie da) to też pójdzie dalej. To jest to samo jakby napisać:

```java
        if(!(StompCommand.SUBSCRIBE.equals(command) || StompCommand.SEND.equals(command))) {
            return;
        }
```